src/app/kvm/components/KVMHeaderForms/ComposeForm/StorageTable/StorageTable.test.tsx

```diff
@@ -201,19 +169,16 @@ describe("StorageTable", () => {
     });
     const state = { ...initialState };
     state.pod.items = [pod];
-    const store = mockStore(state);
-    const wrapper = generateWrapper(store, pod);
+    generateWrapper(state, pod);
 
     // Change the disk size to above 20GB
-    await act(async () => {
-      wrapper.find("input[name='disks[0].size']").simulate("change", {
-        target: { name: "disks[0].size", value: "21" },
-      });
-    });
-    wrapper.update();
+    const diskSizeInput = screen.getByRole("spinbutton", { name: "Size (GB)" });
+    await userEvent.clear(diskSizeInput);
+    await userEvent.type(diskSizeInput, "21");
+
     expect(
-      wrapper.find("StorageTable .p-form-validation__message").text()
-    ).toBe(`Error: Only 20GB available in ${pool.name}.`);
+      screen.getByText("Only 20GB available in", { exact: false })
```

Comment:
[You seem to have an assertion that includes the pool name in another test](https://github.com/canonical/maas-ui/pull/4922/files#diff-cfb4d974fd840373ba1661b8f590564db0555933324709ce14047f59651c2bb3R214), what's the reason you couldn't use it here?

Reply:
Adding this assertion should work now, I had it removed when I was testing for partial matches, I'll add ot now